test/conformance/ingressv2/run.go

```diff
@@ -67,7 +97,21 @@ var alphaTests = map[string]func(t *testing.T){
 // Depending on the options it may test alpha and beta features
 func RunConformance(t *testing.T) {
 
-	for name, test := range stableTests {
+	var stables map[string]func(t *testing.T)
+	var betas map[string]func(t *testing.T)
+	switch test.NetworkingFlags.IngressClass {
```

Comment:
We don't use `--ingressClass` for ingress v2, so I re-used it here. If I should introduce a new flag, I will create it.